<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 4 uliCUT&amp;RUN data analysis | Project Log</title>
  <meta name="description" content="This log records the progress of the summer project." />
  <meta name="generator" content="bookdown 0.17 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 4 uliCUT&amp;RUN data analysis | Project Log" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This log records the progress of the summer project." />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 4 uliCUT&amp;RUN data analysis | Project Log" />
  
  <meta name="twitter:description" content="This log records the progress of the summer project." />
  

<meta name="author" content="LING Ludi" />


<meta name="date" content="2020-05-22" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="download-data.html"/>
<link rel="next" href="chip-seq-data-analysis.html"/>
<script src="libs/header-attrs-2.1/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />











<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">A Minimal Book Example</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Project</a></li>
<li class="chapter" data-level="2" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>2</b> Introduction</a>
<ul>
<li class="chapter" data-level="2.1" data-path="intro.html"><a href="intro.html#transcriptional-factors"><i class="fa fa-check"></i><b>2.1</b> Transcriptional factors</a></li>
<li class="chapter" data-level="2.2" data-path="intro.html"><a href="intro.html#chip-seq"><i class="fa fa-check"></i><b>2.2</b> ChIP-seq</a></li>
<li class="chapter" data-level="2.3" data-path="intro.html"><a href="intro.html#ulicutrun"><i class="fa fa-check"></i><b>2.3</b> uliCUT&amp;RUN</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="download-data.html"><a href="download-data.html"><i class="fa fa-check"></i><b>3</b> Download Data</a>
<ul>
<li class="chapter" data-level="3.1" data-path="download-data.html"><a href="download-data.html#chip-seq-1"><i class="fa fa-check"></i><b>3.1</b> ChIP-seq</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="download-data.html"><a href="download-data.html#samfile"><i class="fa fa-check"></i><b>3.1.1</b> samFile</a></li>
<li class="chapter" data-level="3.1.2" data-path="download-data.html"><a href="download-data.html#download"><i class="fa fa-check"></i><b>3.1.2</b> download</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="download-data.html"><a href="download-data.html#ulicutrun-1"><i class="fa fa-check"></i><b>3.2</b> uliCUT&amp;RUN</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="download-data.html"><a href="download-data.html#samfile-1"><i class="fa fa-check"></i><b>3.2.1</b> samFile</a></li>
<li class="chapter" data-level="3.2.2" data-path="download-data.html"><a href="download-data.html#download-1"><i class="fa fa-check"></i><b>3.2.2</b> download</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="ulicutrun-data-analysis.html"><a href="ulicutrun-data-analysis.html"><i class="fa fa-check"></i><b>4</b> uliCUT&amp;RUN data analysis</a>
<ul>
<li class="chapter" data-level="4.1" data-path="ulicutrun-data-analysis.html"><a href="ulicutrun-data-analysis.html#experimental-design"><i class="fa fa-check"></i><b>4.1</b> Experimental design</a></li>
<li class="chapter" data-level="4.2" data-path="ulicutrun-data-analysis.html"><a href="ulicutrun-data-analysis.html#bedgraph"><i class="fa fa-check"></i><b>4.2</b> bedGraph</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="chip-seq-data-analysis.html"><a href="chip-seq-data-analysis.html"><i class="fa fa-check"></i><b>5</b> ChIP-seq data analysis</a>
<ul>
<li class="chapter" data-level="5.1" data-path="chip-seq-data-analysis.html"><a href="chip-seq-data-analysis.html#qc"><i class="fa fa-check"></i><b>5.1</b> QC</a></li>
<li class="chapter" data-level="5.2" data-path="chip-seq-data-analysis.html"><a href="chip-seq-data-analysis.html#fastq-file"><i class="fa fa-check"></i><b>5.2</b> Fastq file</a></li>
<li class="chapter" data-level="5.3" data-path="chip-seq-data-analysis.html"><a href="chip-seq-data-analysis.html#mm10"><i class="fa fa-check"></i><b>5.3</b> mm10</a></li>
<li class="chapter" data-level="5.4" data-path="chip-seq-data-analysis.html"><a href="chip-seq-data-analysis.html#bowtie2-alignment"><i class="fa fa-check"></i><b>5.4</b> bowtie2 alignment</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Project Log</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ulicutrun-data-analysis" class="section level1" number="4">
<h1><span class="header-section-number">Chapter 4</span> uliCUT&amp;RUN data analysis</h1>
<p>For analyzing the uliCUT&amp;RUN datasets, just begin with the <code>bdg</code>(bedGraph) files.</p>
<p>Steps before getting the bed file (<a href="https://github.com/sarahhainer/uliCUT-RUN">CUT&amp;RUN protocol on GitHub</a>):</p>
<ul>
<li><p>trim reads to 25 bases (while keeping paired)</p></li>
<li><p>split reads using novocraft (also trims off barcode with -l option)</p></li>
<li><p>Align reads using Bowtie2 (align to yeast or to mouse)</p></li>
<li><p>Remove duplicate reads (created through PCR) using <a href="http://broadinstitute.github.io/picard/">Picard</a>.</p></li>
<li><p>Remove low quality reads (MAPQ &lt;10)</p></li>
<li><p>Make Size Distribution file for mm10; Make size distribution for sacCer3, spike in</p></li>
<li><p>Make size classes (<code>1-120 for TF</code>, 150-500 for histones)</p></li>
<li><p>Homer analysis</p></li>
<li><p>makeTagDirectory</p></li>
<li><p>makeUCSCfile</p></li>
<li><p>make aggregation plots or heatmaps over specific anchor location</p></li>
<li><p>call peaks</p></li>
<li><p>find motifs</p></li>
</ul>
<div id="experimental-design" class="section level2" number="4.1">
<h2><span class="header-section-number">4.1</span> Experimental design</h2>
<p><strong>Biological question:</strong></p>
<p>In this project we are only interested in samples from blastocysts, and the goal is to find out which NANOG binding regions are influenced by the depletion of BRG1.</p>
<table>
<caption><span id="tab:unnamed-chunk-1">Table 4.1: </span>Experimental design in CUT &amp; RUN</caption>
<thead>
<tr class="header">
<th align="left">cellType</th>
<th align="left">Experiment</th>
<th align="left">Antibody</th>
<th align="right">Rep</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">blastocysts</td>
<td align="left">EGFPKD</td>
<td align="left">NoAb</td>
<td align="right">4</td>
</tr>
<tr class="even">
<td align="left">blastocysts</td>
<td align="left">EGFPKD</td>
<td align="left">NANOG</td>
<td align="right">4</td>
</tr>
<tr class="odd">
<td align="left">blastocysts</td>
<td align="left">Brg1KD(Smarca4KD)</td>
<td align="left">NoAb</td>
<td align="right">4</td>
</tr>
<tr class="even">
<td align="left">blastocysts</td>
<td align="left">Brg1KD(Smarca4KD)</td>
<td align="left">NANOG</td>
<td align="right">4</td>
</tr>
<tr class="odd">
<td align="left">blastocysts</td>
<td align="left">NanogkD</td>
<td align="left">NoAb</td>
<td align="right">4</td>
</tr>
<tr class="even">
<td align="left">blastocysts</td>
<td align="left">NanogkD</td>
<td align="left">NANOG</td>
<td align="right">4</td>
</tr>
</tbody>
</table>
<div class="figure">
<img src="https://ars.els-cdn.com/content/image/1-s2.0-S0092867419302764-gr6_lrg.jpg" alt="" />
<p class="caption">Hainer, S. J., et al. (2019).</p>
</div>
<p><strong>Hints:</strong></p>
<ol style="list-style-type: decimal">
<li><p>Fig F shows <code>GEFP-KD with NANOG Antibody</code> group does not affect the enrichment of NANOG, so actually these samples are used as the control group in the analysis.</p></li>
<li><p>From EGFP-KD and Smarca4-KD conditions. I think we might tell which NSNOG binding regions are BRG1 related and which are not by analyzing the two samples to find the differential NANOG peaks.</p></li>
</ol>
</div>
<div id="bedgraph" class="section level2" number="4.2">
<h2><span class="header-section-number">4.2</span> bedGraph</h2>
<p><a href="https://m.ensembl.org/info/website/upload/bed.html#bedGraph">BedGraph format</a></p>
<p>The score is placed in column 4, not column 5</p>
<p>Track lines are compulsory, and must include type=bedGraph. Currently the only optional parameters supported by Ensembl are:</p>
<ol style="list-style-type: decimal">
<li><p>name: unique name to identify this track when parsing the file</p></li>
<li><p>description: label to be displayed under the track in Region in Detail</p></li>
<li><p>priority: integer defining the order in which to display tracks, if multiple tracks are defined.</p></li>
<li><p>graphType: either ‘bar’ or ‘points’</p></li>
</ol>
<div class="sourceCode" id="cb1"><pre class="sourceCode bash"><code class="sourceCode bash"><span id="cb1-1"><a href="ulicutrun-data-analysis.html#cb1-1"></a><span class="fu">zcat</span> Brg1KD_1.bedGraph.gz <span class="kw">|</span> <span class="fu">head</span></span>
<span id="cb1-2"><a href="ulicutrun-data-analysis.html#cb1-2"></a><span class="co"># track type=bedGraph name=&quot;Brg1KD_Nanog_1_1-120 Total Tags = 1.16e+05, normalized to 1.00e+07&quot; description=&quot;Brg1KD_Nanog_1_1-120 Total Tags = 1.16e+05, normalized to 1.00e+07&quot; color=123,110,212 visibility=full yLineOnOff=on autoScale=on yLineMark=&quot;0.0&quot; alwaysZero=on graphType=bar maxHeightPixels=128:75:11 windowingFunction=maximum smoothingWindow=off</span></span>
<span id="cb1-3"><a href="ulicutrun-data-analysis.html#cb1-3"></a><span class="co"># chr1    3011588 3011599 41.40</span></span>
<span id="cb1-4"><a href="ulicutrun-data-analysis.html#cb1-4"></a><span class="co"># chr1    3011599 3011692 82.80</span></span>
<span id="cb1-5"><a href="ulicutrun-data-analysis.html#cb1-5"></a><span class="co"># chr1    3011692 3011703 41.40</span></span>
<span id="cb1-6"><a href="ulicutrun-data-analysis.html#cb1-6"></a><span class="co"># chr1    3048182 3048232 41.40</span></span>
<span id="cb1-7"><a href="ulicutrun-data-analysis.html#cb1-7"></a><span class="co"># chr1    3048232 3048286 82.80</span></span></code></pre></div>
<p><a href="https://www.jianshu.com/p/25f7a8967e8a">bedgraph, bed2bam</a></p>
<p>Genome browser tracks were generated from mapped reads using the “makeUCSCfile” command.</p>
<p>Mapped reads were aligned over specific regions using the “annotatePeaks” command to make 20 bp bins over regions of interest and sum the reads within each bin. Peaks were called using parameters similar to those previously described (Skene et al., 2018) but implemented in HOMER using the “findPeaks” command with the following parameters: -style factor &lt; or &gt; histone -P 1 -poisson 0.01 -F 0.5 -L 2 -LP 0.001 -i noab.</p>
<p>Motifs were identified using the “findMotifs” command.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="download-data.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="chip-seq-data-analysis.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/03-uliCUTRUB.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-demo.pdf", "bookdown-demo.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
